processing-chain/pom.xml

```diff
@@ -16,7 +16,7 @@
         <site.basedir>${project.parent.basedir}</site.basedir>
     </properties>
 
-    <artifactId>processing_chain</artifactId>
+    <artifactId>processing-chain</artifactId>
```

Comment:
what about the pom name ? Do we need to rename it ?

Reply:
see my comments in the #41 pull request in orbisgis